src/plot_api/plot_api.js

```diff
@@ -383,6 +383,7 @@ Plotly.plot = function(gd, data, layout, config) {
         drawFramework,
         marginPushers,
         marginPushersAgain,
+        initInteractions,
```

Comment:
for reference (because this diff isn't great), we now have:

![image](https://user-images.githubusercontent.com/6675409/26939962-fa9c73b0-4c46-11e7-8f2b-18497a654cd2.png)